{"classifiers": ["Programming Language :: Python", "Programming Language :: Python :: 2", "Programming Language :: Python :: 2.7", "Programming Language :: Python :: 3", "Programming Language :: Python :: 3.3", "Programming Language :: Python :: 3.4", "Programming Language :: Python :: 3.5", "Programming Language :: Python :: 3.6"], "extensions": {"python.details": {"contacts": [{"email": "Kub-AT@users.noreply.github.com", "name": "Jakub S", "role": "author"}], "document_names": {"description": "DESCRIPTION.rst"}, "project_urls": {"Home": "https://github.com/Kub-AT/django-cache-fallback"}}}, "extras": [], "generator": "bdist_wheel (0.29.0)", "keywords": ["django-cache-fallback", "django", "cache", "multiple", "cache", "fallback", "cache"], "license": "3-clause BSD", "metadata_version": "2.0", "name": "django-cache-fallback", "platform": "any", "run_requires": [{"requires": ["coverage", "django (>=1.8)", "mock", "pytest", "pytest-cov", "python-memcached"]}], "summary": "Django Cache Fallback", "version": "0.2.0"}